changelog.txt

```diff
@@ -1,6 +1,220 @@
+- (#6261) Fix: do not throw an unchecked exception for SpringBoot. Thanks,  @filipelautert
+
+
+## Security, Driver and Other Updates
+
+
+- (#6457) Update jreBundle to latest in install4j @abrackx
+- (#6377) Bump org.testcontainers:testcontainers-bom from 1.20.1 to 1.20.2 @dependabot
+- (#6443) Bump org.junit.platform:junit-platform-suite from 1.11.0 to 1.11.3 @dependabot
+- (#6384) Bump org.apache.maven.plugins:maven-javadoc-plugin from 3.10.0 to 3.10.1 @dependabot
+- (#6387) Bump org.codehaus.gmavenplus:gmavenplus-plugin from 3.0.2 to 4.0.1 @dependabot
+- (#6400) Bump org.apache.maven.plugins:maven-surefire-plugin from 3.5.0 to 3.5.1 @dependabot
+- (#6445) Bump org.junit.jupiter:junit-jupiter from 5.11.0 to 5.11.3 @dependabot
+- (#6408) Bump actions/cache from 4.0.2 to 4.1.1 @dependabot
+- (#6444) Bump junit-jupiter.version from 5.11.0 to 5.11.3 @dependabot
+- (#6401) Bump org.apache.maven.plugins:maven-failsafe-plugin from 3.5.0 to 3.5.1 @dependabot
+- (#6399) Bump codex-/return-dispatch from 1 to 2 @dependabot
+- (#6368) Bump org.xerial:sqlite-jdbc from 3.46.1.0 to 3.46.1.3 @dependabot
+- (#6319) Bump groovy.version from 4.0.22 to 4.0.23 @dependabot
+- (#6337) Bump commons-io:commons-io from 2.16.1 to 2.17.0 @dependabot
+- (#6290) Bump org.yaml:snakeyaml from 2.2 to 2.3 @dependabot
+- (#6274) Bump org.jboss.weld.se:weld-se-core from 5.1.2.Final to 5.1.3.Final @dependabot
+- (#6276) Bump org.apache.maven.plugins:maven-surefire-plugin from 3.3.1 to 3.5.0 @dependabot
+- (#6275) Bump org.apache.maven.plugins:maven-failsafe-plugin from 3.3.1 to 3.5.0 @dependabot
+- (#6273) Bump org.apache.maven.plugins:maven-plugin-plugin from 3.13.1 to 3.15.0 @dependabot
+- (#6284) Bump actions/setup-python from 5.1.1 to 5.2.0 @dependabot
+- (#6282) Bump ant.version from 1.10.14 to 1.10.15 @dependabot
+- (#6283) Bump org.apache.commons:commons-lang3 from 3.16.0 to 3.17.0 @dependabot
+- (#6272) Bump org.apache.maven.plugins:maven-javadoc-plugin from 3.8.0 to 3.10.0 @dependabot
+- (#6247) Bump com.microsoft.sqlserver:mssql-jdbc from 12.6.1.jre8 to 12.8.1.jre8 @dependabot
+- (#6246) Bump org.postgresql:postgresql from 42.7.3 to 42.7.4 @dependabot
+- (#6221) Bump org.xerial:sqlite-jdbc from 3.46.0.0 to 3.46.1.0 @dependabot
+- (#6220) Bump org.apache.maven.plugins:maven-install-plugin from 3.1.2 to 3.1.3 @dependabot
+- (#6219) Bump org.apache.maven.plugins:maven-deploy-plugin from 3.1.2 to 3.1.3 @dependabot
+- (#6213) Bump targetMavenVersion from 3.9.8 to 3.9.9 @dependabot
+- (#6202) Bump org.junit.jupiter:junit-jupiter from 5.10.3 to 5.11.0 @dependabot
+- (#6201) Bump org.junit.platform:junit-platform-suite from 1.10.3 to 1.11.0 @dependabot
+- (#6200) Bump spring.version from 5.3.37 to 5.3.39 @dependabot
+- (#6245) Bump actions/delete-package-versions from 3 to 5 @dependabot
+- (#6199) Bump junit-jupiter.version from 5.10.3 to 5.11.0 @dependabot
+
+
+
```

Comment:
Except the full changelog you can remove everything below this line .

Reply:
Updated!